src/zope/app/container/browser/tests/test_contents_functional.py

```diff
@@ -348,15 +350,15 @@ def test_copy_then_delete_with_unicode_name(self):
             basic='mgr:mgrpw',
             form={
                 'ids:list' : u'voil\xe0'.encode('utf-8'),
```

Comment:
I really hate to do this, but there's a space in front of a `:` here too.

I wouldn't bring it up, except this ` :` shows up in the git diff as an added line.